src/main/java/uk/gov/hmcts/reform/professionalapi/controller/request/OrganisationIdentifierValidatorImpl.java

```diff
@@ -68,4 +78,12 @@ public void validateOrganisationIsActive(Organisation existingOrganisation) {
             throw new EmptyResultDataAccessException(1);
         }
     }
+
+    public void validateOrganisationExists(String orgId) {
+        if (null == organisationService.getOrganisationByOrgIdentifier(orgId)) {
```

Comment:
I think validateOrganisationIdentifier already there and remove this method.

Reply:
This is validating whether the Organisation Identifier provided, while ti may be "valid" actually belongs to an organisation. This was part of the issue raised by ITHC.